src/styles/components/_component-grid.scss

```diff
@@ -284,30 +288,37 @@ $breakpoint-accordion-large-755: 47rem;
       &::before {
         @include doodle;
 
-        width: 25%;
-        height: 25%;
+        width: 20%;
+        height: 20%;
```

Comment:
and here you may wish to have a min-height / min-width in rem for these values

Reply:
Good point, yet again!

Updated here: https://github.com/sparkbox/accessible-components/pull/89/commits/fdbd0fa7e892b31ebf8a8abec1ce0fb026068b25